src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.ts

```diff
@@ -19,7 +19,7 @@ import { DotHttpErrorManagerService } from '../../../../api/services/dot-http-er
  * @implements {Resolve<DotRenderedPageState>}
  */
 @Injectable()
-export class DotEditContentResolver implements Resolve<DotRenderedPageState> {
+export class DotEditPageResolver implements Resolve<DotRenderedPageState> {
```

Comment:
Is not resolving only content now... it resolver the whole page: content and layout or template.